src/components/Steps/Step.js

```diff
@@ -0,0 +1,80 @@
+import * as React from 'react'
+import cx from 'classnames'
+
+import checkmark from './../../assets/checkmark.svg'
+
+import styles from './Steps.module.scss'
+
+const Step = ({
+  className,
+  active,
+  status = 'wait',
+  stepNumber,
+  disabled,
+  description,
+  title,
+  icons,
+  stepIndex,
+  onStepClick,
+  onClick,
+  ...restProps
+}) => {
+  const handleClick = (...args) => {
+    if (onClick) {
+      onClick(...args)
+    }
+
+    onStepClick(stepIndex)
+  }
+
+  const renderIconNode = () => {
+    if (icons && icons.process && status === 'process') {
+      return icons.process
+    }
+    if (status === 'finish') {
+      return <img src={checkmark} alt='Check' />
+    }
+    return <span className={styles.stepsIcon}>{stepNumber}</span>
+  }
+
+  const classString = cx(
+    styles.stepsItem,
+    {
+      [styles.stepsItemWait]: status === 'wait' && !disabled,
+      [styles.stepsItemProcess]: status === 'process',
+      [styles.stepsItemFinish]: status === 'finish',
+      [styles.stepsItemDisabled]: disabled
+    },
+    className
+  )
+
+  const accessibilityProps = {}
+  if (onStepClick && !disabled) {
+    accessibilityProps.role = 'button'
+    accessibilityProps.tabIndex = 0
+    accessibilityProps.onClick = handleClick
+  } else if (onClick && !disabled) {
+    accessibilityProps.role = 'button'
+    accessibilityProps.tabIndex = 0
+    accessibilityProps.onClick = onClick
+  }
+
+  return (
+    <div {...restProps} className={classString}>
+      <div {...accessibilityProps} className={styles.stepsItemContainer}>
+        <div className={styles.stepsItemTail} />
```

Comment:
@DmitriVanGuard Do we really need to make so long names like stepsItemTail, stelsItemContentContainer? I mean we are using modules, we can use itemTail and it will be unique anyway. Or I am not right already.
cc @Woafflation